cmd/kops/promote_keypair.go

```diff
@@ -162,17 +162,17 @@ func promoteKeypair(out io.Writer, name string, keypairID string, keyStore fi.CA
 	}
 
 	if keypairID == "" {
-		highestTrustedId := big.NewInt(0)
+		highestCandidateId := big.NewInt(0)
```

Comment:
Since we're searching for a private key that's neither distrusted nor lacking a certificate, I figured I'd change the name here to be more general.